openedx/core/lib/api_utils.py

```diff
@@ -0,0 +1,73 @@
+"""Helper functions to get data from APIs"""
+from __future__ import unicode_literals
+import logging
+
+from django.core.cache import cache
+from edx_rest_api_client.client import EdxRestApiClient
+
+from openedx.core.lib.token_utils import get_id_token
+
+
+log = logging.getLogger(__name__)
+
+
+def get_api_data(api_config, user, api_name, resource, querystring=None, use_cache=False, cache_key=None):
```

Comment:
A more precise name for this function might be `get_edx_api_data`. Likewise, a more precise name for the module would be `edx_api_utils`. This makes it clear that these utilities are meant for communicating with edX APIs.